MicrosoftFluentUI.podspec

```diff
@@ -329,6 +329,12 @@ xcodebuild ${XCODEBUILDPARAMS} -project ${PROJECT_FILE_PATH} -target "MicrosoftF
     avatarview_mac.source_files = ["macos/FluentUI/AvatarView/**/*.{swift,h}"]
   end
 
+  s.subspec 'BadgeView_mac' do |badgeview_mac|
+    badgeview_mac.platform = :osx
+    badgeview_mac.dependency 'MicrosoftFluentUI/Core_mac'
+    badgeview_mac.source_files = ["macos/FluentUI/BadgeView/**/*.{swift,h}"]
```

Comment:
This seems to be the reason you have pod lib lint failing in the validation checks.
The pattern is looking for macos/FluentUI/**BadgeView** while you added files in the macos/FluentUI/**Badge** folder.